src/Command.js

```diff
@@ -29,17 +29,17 @@ module.exports = BotModule.extend({
             }
 
             if (command === this.commandName || this.commandAliases.includes(command)) {
-                let timeOut = this.config.messageRemoveDelay;
+                let timeout = this.config.messageRemoveDelay;
                 let promise = this.processMessage(message, tokens);
 
                 if (!this.shouldDeleteMessage) {
                     return;
                 }
 
-                message.delete(timeOut).catch(err => {/*do nothing*/});
+                message.delete({timeout}).catch(() => {/*do nothing*/});
```

Comment:
Is this syntax correct?  It looks like you're creating an object with an un-named member called "timeout"?

Reply:
Yeah, the new syntax of deleting messages requires an object, not a single value. See this section of the upgrade guide: https://discordjs.guide/additional-info/changes-in-v12.html#message-delete.

In the case of ES6, this is not an unnamed property but a shortcut for `{timeout: timeout}`. It's equivalent to then simply say `{timeout}`.